test.nix

```diff
@@ -1,7 +1,7 @@
 {nixpkgs ? import <nixpkgs> {}}:
 with nixpkgs;
 let
-  compilers = [ "ghc802" "ghc822" "ghc843" "ghc861" ];
+  compilers = [ "ghc802" "ghc822" "ghc843" "ghc861" "ghc881" ];
```

Comment:
IDK how to test this. But it compiles fine with `stack` on the `nightly-2019-09-27` resolver (which uses `ghc-8.8.1`).